etc/DependencyInstaller.sh

```diff
@@ -120,11 +120,13 @@ _installUbuntuDev() {
         automake \
         autotools-dev \
         bison \
+	doxygen \
         flex \
         clang \
         g++ \
         gcc \
         git \
+	graphviz \
```

Comment:
Why add these packages? Also, if we add for ubuntu we should add for other all the OS.

Reply:
These packages were missing and needed to be installed by hand to get the docker container to work. Will fix for other OSes.